electron/main/index.ts

```diff
@@ -182,6 +221,19 @@ async function createWindow() {
   ipcMain.handle('getChain', (event, coin) => {
     return sdk.coin(coin).getChain();
   });
+
+  ipcMain.handle('getUser', () => {
+    try {
+      return sdk.me();
+    } catch (e: unknown) {
+      return new Error(handleSdkError(e));
+    }
+  });
+
+  ipcMain.handle('isSdkAuthenticated', async event => {
+    const sdkJson = sdk.toJSON();
+    return Promise.resolve(sdkJson.user || sdkJson.token);
```

Comment:
if you mark a function as `async` you don't need to wrap it in a Promise.resolve due to the nature of how async functions work, that happens implicitly.